json-path/src/main/java/com/jayway/jsonpath/internal/path/CompiledPath.java

```diff
@@ -142,4 +147,17 @@ public boolean isFunctionPath() {
     public String toString() {
         return root.toString();
     }
+    
+    /**
+     * Fetches the relational expression values from this path.
+     *
+     * @return EvaluationContext containing results of evaluation
+     */
+    @SuppressWarnings("unchecked")
+	@Override
+    public ArrayList<SimpleEntry<String, String>> getRelationalExprValues(){
+    	ArrayList<SimpleEntry<String, String>> valuesMap = new ArrayList<SimpleEntry<String, String>>();
+    	root.next().getRelationalExprValues(valuesMap);
```

Comment:
try to use our coding standards like @Nonnull , final for variables that are final and not change

Reply:
Done